client/src/javascript/components/torrent-list/TorrentList.js

```diff
@@ -252,6 +253,15 @@ class TorrentListContainer extends React.Component {
         defaultMessage: 'Torrent Details'
       })
     }, {
+      action: 'torrent-download-tar',
+      clickHandler: (action, event) => {
+        clickHandler(action, event, torrent);
+      },
+      label: this.props.intl.formatMessage({
+        id: 'torrents.list.context.download',
+        defaultMessage: 'Download .tar'
```

Comment:
What do you think about changing this label to `Download`? If it's a single file torrent, then it won't be archived.

Reply:
I don't mind.

`Download` is confusing anyway,  as we have `Set Download Location`...

Maybe if we changed that `Set Torrent Data Location` it would be OK?

Reply:
Yes, `Set Torrent Data Location` is a lot more reasonable. Good idea!